package.json

```diff
@@ -7,22 +7,22 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^3.4.0",
     "@typescript-eslint/parser": "^3.4.0",
-    "concurrently": "^5.2.0",
     "eslint": "^7.2.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.2",
     "prettier": "^2.0.5",
-    "typescript": "^3.8.3"
+    "typescript": "^3.9.6"
   },
   "dependencies": {
-    "@angular/forms": "^10.0.2",
     "@angular/animations": "^10.0.2",
     "@angular/common": "^10.0.2",
     "@angular/core": "^10.0.2",
+    "@angular/forms": "^10.0.2",
     "@angular/platform-browser": "^10.0.2",
     "@angular/router": "^10.0.2",
+    "concurrently": "^5.2.0",
```

Comment:
why the switch from devDep to dep

Reply:
This was an accident -- Will change sometime.